SAKPaaS/src/app/components/location-panel/location-panel.component.scss

```diff
@@ -2,29 +2,30 @@
   z-index: 3;
   display: flex;
   flex-direction: column;
-  padding: 10px 10px 0 10px;
+  padding: 7px 10px 10px 6px;
   max-width: 500px;
   width: 100%;
   background-color: white;
   height: auto;
   position: fixed;
   top: 35vh;
-  bottom: 54px;
```

Comment:
You should not remove that.

![image](https://user-images.githubusercontent.com/15220797/81192159-2d01c080-8fba-11ea-847f-64a04f13b269.png)

Reply:
At least not in the standard view. You could always change it to `bottom: 0` in `@media`.